sqlstore/src/main/resources/db/migration/V8080__shortname.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE Project CHANGE COLUMN shortName shortName VARCHAR(255);
```

Comment:
The edit project page uses the length of this column to dictate how many characters are allowed in the input field. This will fail.

Reply:
Wait, so we should change the page to get length information from the naming scheme rather than the DB (or probably some min of the two)?

Reply:
I'd be up for that. What do you think, @djcooke ?

Reply:
Makes sense to get the length from the validator instead